docs/eos-learn/accounts-and-permissions.md

```diff
@@ -28,7 +28,7 @@ Any given account can define a mapping between any of its named permissions and
 
 In other words each account's permission can be linked to an authority table used to determine whether a given action authorization can be satisfied
 
-To get more information about these concepts, see [accounts and permissions documentation](https://developers.eos.io/welcome/latest/protocol/accounts_and_permissions).
+To get more information about these concepts, see [accounts and permissions documentation](https://developers.eos.io/welcome/v2.0/protocol-guides/accounts_and_permissions).
```

Comment:
shuldn't we use the `latest` version?

Reply:
in other words the correct link to use should be : https://developers.eos.io/welcome/latest/protocol-guides/accounts_and_permissions